java/code/webapp/WEB-INF/pages/common/fragments/kickstart/schedule/advanced.jspf

```diff
@@ -7,47 +7,46 @@
 
 <head>
 <meta http-equiv="Pragma" content="no-cache">
+</head>
 
-<script language="javascript">
-  function moveNext() {
-    var form = document.getElementsByName("kickstartScheduleWizardForm")[0];
-    form.submit();
-  }
+<body>
+    <script language="javascript">
+        function moveNext() {
+            var form = document.getElementsByName("kickstartScheduleWizardForm")[0];
```

Comment:
It might be interesting to handle exceptions in some of those methods, like if the elements array is empty.

Reply:
same as above